back/app/models/idea_status.rb

```diff
@@ -13,14 +13,16 @@
 #  updated_at           :datetime         not null
 #  description_multiloc :jsonb
 #  ideas_count          :integer          default(0)
+#  participation_method :string           default("ideation"), not null
 #
 class IdeaStatus < ApplicationRecord
```

Comment:
I did a cleanup of the idea status model and moved some logic to the controller level, which felt more appropriate (for example, many specs don't need a proposed status). The logic to ensure that there is a proposed status had to be changed, because there are now two different proposed statuses.